hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -710,6 +710,12 @@
       sw_images: ["//sw/device/tests/sim_dv:spi_host_tx_rx_test:1:new_rules"]
       en_run_modes: ["sw_test_mode_test_rom"]
     }
+    {
+      name: chip_sw_spi_device_pinmux_sleep_retention
```

Comment:
Please add this test to the list of `tests` for `chip_sw_spi_device_output_when_disabled_or_sleeping` in `hw/top_earlgrey/data/chip_spi_device_testplan.hjson`:
https://github.com/lowRISC/opentitan/blob/c04cc5d074d9acefd052feb8a4fd6de6bfa0faab/hw/top_earlgrey/data/ip/chip_spi_device_testplan.hjson#L134-L169

Reply:
Done

Reply:
Done - I've only added the spi-device features for the registers configured in SW, since this is more a pinmux test than a spi-device one